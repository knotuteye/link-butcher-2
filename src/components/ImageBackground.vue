<template>
  <div class="background"></div>
</template>

<script lang="ts">
import { Component, Prop, Vue } from 'vue-property-decorator'

@Component
export default class ImageBackground extends Vue {}
</script>

<style scoped>
div.background {
  min-height: 100vh;
  width: 100vw;
  background-image: url('../assets/backImg.jpg');
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
}
</style>
