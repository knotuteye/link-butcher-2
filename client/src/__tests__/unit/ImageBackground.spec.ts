import { shallowMount } from '@vue/test-utils'
import ImageBackground from '../../components/ImageBackground.vue'

describe('ImageBackground.vue', () => {
  test('is rendered as designed', () => {
    const wrapper = shallowMount(ImageBackground)
    expect(wrapper.element).toMatchSnapshot()
  })
})
