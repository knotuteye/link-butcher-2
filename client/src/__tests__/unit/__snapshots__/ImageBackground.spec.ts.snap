// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageBackground.vue is rendered as designed 1`] = `
<div
  class="background"
/>
`;
